plugin.program.autocompletion/addon.xml

```diff
@@ -16,7 +16,7 @@
         <description lang="es_ES">Autocompletar para el teclado virtual (necesita soporte de skin)</description>
         <platform>all</platform>
         <license>GPL-2.0-or-later</license>
-        <source>https://www.github.com/finkleandeinhorn/plugin.program.autocompletion</source>
+        <source>https://github.com/finkleandeinhorn/script.module.autocompletion</source>
```

Comment:
Intended?

Reply:
Whoops, nope. Working on these two back to back, and you keep catching me 😅